src/databricks/labs/ucx/assessment/export.py

```diff
@@ -0,0 +1,51 @@
+import logging
+from pathlib import Path
+
+from databricks.labs.blueprint.tui import Prompts
+
+from databricks.labs.ucx.config import WorkspaceConfig
+from databricks.labs.lsql.backends import SqlBackend
+from databricks.labs.lsql.dashboards import DashboardMetadata
+
+logger = logging.getLogger(__name__)
+
+
+class AssessmentExporter:
+    # File and Path Constants
+    _EXPORT_FILE_NAME = "ucx_assessment_results.zip"
+
+    def __init__(self, sql_backend: SqlBackend, config: WorkspaceConfig):
+        self._sql_backend = sql_backend
+        self._config = config
+
+    def export_results(self, prompts: Prompts):
+        """Main method to export results to CSV files inside a ZIP archive."""
+        project_root = Path(__file__).resolve().parents[3]
+        queries_path_root = project_root / f"labs/ucx/queries/assessment"
+        valid_queries = {subdir.name for subdir in queries_path_root.iterdir() if subdir.is_dir()}
+
+        export_path = Path(
+            prompts.question(
+                "Choose a path to save the UCX Assessment results",
+                default=Path.cwd().as_posix(),
+                validate=lambda p_: Path(p_).exists(),
+            )
+        )
+
+        query_choice = prompts.question(
```

Comment:
there's `prompts.choice()` for selecting from a list. but why do we need this choice? can't we just export the main assessment dashboard out?

Reply:
The code defaults to running queries available in Main. But we also wanted to extend the capability to run other queries in folders 'estimates', 'interactive', 'azure', 'main' under labs/ucx/queries/assessment

Reply:
@JCZuurmond suggested adding this capability. PFA for previous comments suggesting this need.
![image (5)](https://github.com/user-attachments/assets/f472db9e-45c2-445b-9613-638af3f2d2ee)

Reply:
Using prompts.question() allows us to set 'main' as the default input, which isn't possible with prompts.choice().
<img width="745" alt="image" src="https://github.com/user-attachments/assets/d244934c-685f-4cb9-a4c5-59471d65a06e">
<img width="844" alt="image" src="https://github.com/user-attachments/assets/6616bf25-b1fa-4ea6-b66a-f1ea90204145">

Reply:
do we really need this? can we simplify?